packages/tools/client-debugger/client-debugger-view/src/components/DataObjectsView.tsx

```diff
@@ -2,33 +2,35 @@
  * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
  * Licensed under the MIT License.
  */
+import { HasContainerId } from "@fluid-tools/client-debugger";
 import React from "react";
 
-import { HasClientDebugger } from "../CommonProps";
 import { SharedObjectRenderOptions } from "../RendererOptions";
 import { DynamicDataView } from "./data-object-views";
 
 /**
  * {@link DataObjectsView} input props.
  */
-export interface DataObjectsViewProps extends HasClientDebugger {
+export interface DataObjectsViewProps extends HasContainerId {
 	/**
 	 * {@inheritDoc RendererOptions}
 	 */
 	renderOptions: SharedObjectRenderOptions;
+
+	containerData: unknown;
```

Comment:
`DataObjectsView` will be querying for its data via the MessageRelay, right? Is this needed?

Reply:
Or is this a temporary thing to preserve the existing behavior?

Reply:
The prop will go away, yep. I haven't gotten to this view to make the full update so just needed it to build in the meantime.